storybook/stories/chart/radarchart/RadarChart.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Radar, RadarChart, ResponsiveContainer } from '../../../../src';
+import { pageData } from '../../data';
+
+export default {
+  title: 'Recharts/Chart/Radar Chart',
+  component: RadarChart,
+};
+
+export const Simple = {
+  render: (args: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <RadarChart data={args.data} width={600} height={500} outerRadius={150}>
```

Comment:
```suggestion
        <RadarChart {...args}>
```

The controls will only work, if you actually pass the args to the component. This must be done in every example.